src/controller/python/chip/clusters/Command.py

```diff
@@ -117,20 +128,91 @@ def handleError(self, status: Status, chipError: PyChipError):
             self._handleError, status, chipError, None
         )
 
+    def handleDone(self):
+        ctypes.pythonapi.Py_DecRef(ctypes.py_object(self))
+
+
+class AsyncBatchCommandsTransaction:
+    def __init__(self, future: Future, eventLoop, expectTypes: List[Type]):
+        self._event_loop = eventLoop
+        self._future = future
+        self._expect_types = expectTypes
+        default_im_failure = chip.interaction_model.InteractionModelError(
+            chip.interaction_model.Status.Failure)
+        self._responses = [default_im_failure] * len(expectTypes)
```

Comment:
This may confuse clients in thinking "FAILURE" was the result of the command. Should we not use something like TIMEOUT at least?

Reply:
This will be changed to `NO_COMMAND_RESPONSE` after https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8645, lands and I get the corresponding IM changes into the SDK.

But sure for now I will switch this to TIMEOUT